src/psyclone/tests/psyir/backend/fortran_routine_test.py

```diff
@@ -367,6 +367,11 @@ def test_fw_routine_prefixes(fortran_reader, fortran_writer):
     end module test'''
     container = fortran_reader.psyir_from_source(code)
     output = fortran_writer(container)
+    routine = container.children[0].children[0]
```

Comment:
More robust to do `routine=container.walk(Routine)[0]`

Reply:
Fixed all of these.